extensions/resource-deployment/src/services/resourceTypeService.ts

```diff
@@ -248,42 +246,12 @@ export class ResourceTypeService implements IResourceTypeService {
 	}
 
 
-	public startDeployment(resourceType: ResourceType, provider: DeploymentProvider): void {
-		const self = this;
-		if (instanceOfDialogDeploymentProvider(provider)) {
-			const dialog = new DeploymentInputDialog(this.notebookService, this.platformService, this.toolsService, provider.dialog);
-			dialog.open();
-		} else if (instanceOfNotebookDeploymentProvider(provider)) {
-			this.notebookService.openNotebook(provider.notebook);
-		} else if (instanceOfDownloadDeploymentProvider(provider)) {
-			const taskName = localize('resourceDeployment.DownloadAndLaunchTaskName', "Download and launch installer, URL: {0}", provider.downloadUrl);
-			azdata.tasks.startBackgroundOperation({
-				displayName: taskName,
-				description: taskName,
-				isCancelable: false,
-				operation: op => {
-					op.updateStatus(azdata.TaskStatus.InProgress, localize('resourceDeployment.DownloadingText', "Downloading from: {0}", provider.downloadUrl));
-					self.download(provider.downloadUrl).then(async (downloadedFile) => {
-						op.updateStatus(azdata.TaskStatus.InProgress, localize('resourceDeployment.DownloadCompleteText', "Successfully downloaded: {0}", downloadedFile));
-						op.updateStatus(azdata.TaskStatus.InProgress, localize('resourceDeployment.LaunchingProgramText', "Launching: {0}", downloadedFile));
-						await this.platformService.runCommand(downloadedFile, { sudo: true });
-						op.updateStatus(azdata.TaskStatus.Succeeded, localize('resourceDeployment.ProgramLaunchedText', "Successfully launched: {0}", downloadedFile));
-					}, (error) => {
-						op.updateStatus(azdata.TaskStatus.Failed, error);
-					});
-				}
-			});
-		} else if (instanceOfWebPageDeploymentProvider(provider)) {
-			vscode.commands.executeCommand('vscode.open', vscode.Uri.parse(provider.webPageUrl));
-		} else if (instanceOfCommandDeploymentProvider(provider)) {
-			vscode.commands.executeCommand(provider.command);
-		} else {
-			const wizard = new ResourceTypeWizard(resourceType, provider, new KubeService(), new AzdataService(this.platformService), this.notebookService, this.toolsService, this.platformService);
-			wizard.open();
-		}
+	public startDeployment(resourceType: ResourceType): void {
```

Comment:
What happenned to all the non ResourceTypeWizard deployments, whwere did they end up?